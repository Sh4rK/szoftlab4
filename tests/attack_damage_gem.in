loadMap test.map
loadMission attack.mission
buildTower 7 1
enchant red 7 1
listTowers
step 34
listEnemies
step 1
listEnemies

exit
