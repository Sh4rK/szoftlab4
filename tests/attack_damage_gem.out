(7.0;1.0) red
1 75.0 (8.6;8.6)
1 45.0 (8.8;8.8)
